src/02-data-types.js

```diff
@@ -127,7 +156,13 @@ function isOdd(num) {}
  *  isTruthy(null);
  *  //> false
  */
-function isTruthy(val) {}
+ function isTruthy(val) {
+  if (val === null || val === 0 || val === "" || val === undefined || val === NaN){
```

Comment:
I actually like that you've written out these conditions.  Can we shorten this in anyway?